WordPress/src/main/java/org/wordpress/android/ui/photopicker/PhotoPickerAdapter.java

```diff
@@ -172,6 +174,14 @@ boolean isVideoUri(Uri uri) {
         return index > -1 && getItemAtPosition(index).isVideo;
     }
 
+    void setAllowMultiSelect(boolean allow) {
+        mAllowMultiSelect = allow;
+    }
+
+    void setShowPhotosOnly(boolean value) {
```

Comment:
Right now the activity only enables selecting photos - [see here](https://github.com/wordpress-mobile/WordPress-Android/pull/5563/files#diff-0e5cc7e92c7282b3a62c30b8fe2b4051R75). But we should definitely change the title if/when we enabling selecting videos as well.